src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -227,20 +240,87 @@ const useGroupInfo = (channelId = null) => {
 
   const onRemoveUser = async () => {
     setOpenModal(false);
+    const responseChannelData = await removeMemberGroup({
+      channelId,
+      targetUserId: selectedUser?.userId
+    });
+
     try {
-      const result = await channel.removeMembers([selectedUser.user_id || selectedUser.userId]);
-      const updateParticipant = newParticipant.filter(
-        (participant) => participant.user_id !== (selectedUser.user_id || selectedUser.userId)
+      const {channelName} = getChannelListInfo(
+        responseChannelData.data,
+        signedProfileId,
+        anonProfileId
       );
-      setNewParticipant(updateParticipant);
-      updateMemberName(result.members);
-      setNewParticipant(result.members);
-      setParticipants(result.members, groupPatchDispatch);
+      const channelList = await ChannelListSchema.getSchemaById(localDb, channelId);
+      channelList.name = channelName;
+      await channelList.save(localDb);
+
+      setSelectedChannel(channelList);
+
+      await UserSchema.deleteByUserId(localDb, selectedUser?.userId, channelId);
     } catch (e) {
-      if (__DEV__) {
-        console.log(e, 'error');
-      }
+      console.log('error on memberSchema');
+      console.log(JSON.stringify(e));
+      console.log(e);
+    }
+
+    refresh('channelList');
+    refresh('chat');
+    refresh('channelInfo');
+
+    setTimeout(() => {
+      navigation.navigate('SignedChatScreen');
+    }, 500);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onRemoveUser` function has been refactored to use the `removeMemberGroup` service and update the local database accordingly. Error handling is also included. However, it's not a good practice to log errors directly to the console in production code. Instead, consider using a logging service that can provide more context about the error and help with debugging.
```diff
- console.log('error on memberSchema');
- console.log(JSON.stringify(e));
- console.log(e);
+ // Consider replacing the above lines with a logging service
+ logger.error('Error on memberSchema', e);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->